psql/functions/create_owner_by_login.sql

```diff
@@ -1,96 +1,63 @@
-CREATE FUNCTION app_private.create_owner_by_login(
-  service app_public.git_service,
-  service_id text,
-  username username,
-  name text,
-  email email,
-  oauth_token text
-) RETURNS json AS $$
-  DECLARE
-    profile_picture text;
-  BEGIN
-    profile_picture := '';
-    IF $1='github' THEN
-      profile_picture := CONCAT('https://avatars.githubusercontent.com/u/', $2);
-    END IF;
-    RETURN app_private.create_owner_by_login(
-      $1, $2, $3, $4, $5, $6, profile_picture, TRUE);
-  END;
-$$ LANGUAGE plpgsql VOLATILE SET search_path FROM CURRENT;
-
 CREATE FUNCTION app_private.create_owner_by_login(
   service app_public.git_service,
   service_id text,
   username username,
   name text,
   email email,
   oauth_token text,
-  profile_image_url text
-) RETURNS json AS $$
-  BEGIN
-    RETURN app_private.create_owner_by_login(
-      $1, $2, $3, $4, $5, $6, $7, TRUE);
-  END;
-$$ LANGUAGE plpgsql VOLATILE SET search_path FROM CURRENT;
-
-CREATE FUNCTION app_private.create_owner_by_login(
-    service app_public.git_service,
-    service_id text,
-    username username,
-    name text,
-    email email,
-    oauth_token text,
-    profile_image_url text,
-    is_user boolean
+  profile_image_url text default null
 ) RETURNS json AS $$
   DECLARE _owner_uuid uuid DEFAULT NULL;
   DECLARE _owner_vcs_uuid uuid DEFAULT NULL;
   DECLARE _token_uuid uuid DEFAULT NULL;
   BEGIN
+    IF profile_image_url IS NULL AND $1 = 'github' THEN
+      profile_image_url = CONCAT('https://avatars.githubusercontent.com/u/', $2);
```

Comment:
According to your other changes, this would be:
```suggestion
      profile_image_url = CONCAT('https://avatars.githubusercontent.com/u/', create_owner_by_login.service_id);
```